aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1840,7 +1800,9 @@ static Tensor& linalg_norm_out_impl(Tensor& result, const Tensor& self, optional
     // 'ord' is int or None
     std::vector<int64_t> dim_ = opt_dim.has_value() ? opt_dim.value().vec() : make_dim_list(ndim);
     if (!opt_num_ord.has_value() || dim_.size() == 1) {
-      _linalg_norm_vector_out(result, self, opt_num_ord, dim_, keepdim, opt_dtype);
+      Tensor result_ = at::linalg_vector_norm(self, opt_num_ord, opt_dim, keepdim, opt_dtype);
+      resize_output(result, result_.sizes());
+      result.copy_(result_);
```

Comment:
Per our offline discussion, let's add a comment here with a link to an issue for updating the out= copy behavior (to be safe castable and check the device)